Assets/Mirage/Runtime/CharacterSpawner.cs

```diff
@@ -74,7 +74,7 @@ void OnDestroy()
             }
         }
 
-        private void OnClientConnected(INetworkPlayer player)
+        protected internal void OnClientConnected(INetworkPlayer player)
```

Comment:
can this just be `internal`?
```suggestion
        internal void OnClientConnected(INetworkPlayer player)
```